src/index.html

```diff
@@ -7,7 +7,119 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+
+    <div class="stars__blocks">
```

Comment:
```suggestion
  <body class="body">
    <div class="stars__blocks">
```
Don't leave newlines between parent and child elements